### PR TITLE
Fix random quotation format

### PR DESCRIPTION
**Pull request by jpaints3**
There was an earlier commit that got caught in the previous pull request. I time allows can you check that one out too? Thanks!

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/StringServlet.java [hunk 1]
```diff
@@ -19,9 +19,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //create a new array list and input quoataions.
         ArrayList<String> quotations = new ArrayList<String>();
 
-        quotations.add("<p>The past is not dead. In fact, it's not even past.  -William Faulkner</p>");
-        quotations.add("<p>Education is the passport to the future, for tomorrow belongs to those who prepare for it today. -Malcolm X</p>");
-        quotations.add("<p>Don't count the days, make the days count. -Muhammad Ali</p>");
+        quotations.add("The past is not dead. In fact, it's not even past.  -William Faulkner");
+        quotations.add("Education is the passport to the future, for tomorrow belongs to those who prepare for it today. -Malcolm X");
```
Comment (josueetcom) [inline portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: "Java code has a column limit of 100 characters." - https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

File: portfolio/src/main/webapp/fetcher.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 //quick fetch function 
-async function strFetch(){
-    const gson = new Gson();
-    const servletText = (await fetch('/strservlet')).text(); //get the servlet text
+async function strFetch(){  
+  const servletText = await (await fetch('/strservlet')).json(); //get the servlet text as json
 
-    const servletJSON = gson.toJSON(servletText) 
+  let randomQuote = ""
+  let randomIdx = Math.floor(Math.random() * servletText.length)
```
Comment (josueetcom) [inline portfolio/src/main/webapp/fetcher.js]: "Every statement must be terminated with a semicolon. Relying on automatic semicolon insertion is forbidden."

https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required

File: portfolio/src/main/webapp/fetcher.js [hunk 3]
```diff
@@ -1,9 +1,10 @@
...
+  randomQuote = await servletText[randomIdx]; //choose a quote from a random index
```
Comment (josueetcom) [inline portfolio/src/main/webapp/fetcher.js]: It looks like there's nothing to `await` on here?
```suggestion
  randomQuote = servletText[randomIdx]; //choose a quote from a random index
```

File: portfolio/src/main/webapp/fetcher.js (hunk 2 above)
Comment (josueetcom) [inline portfolio/src/main/webapp/fetcher.js]: "Use `const` by default, unless a variable needs to be reassigned."

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

File: portfolio/src/main/webapp/fetcher.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
 //quick fetch function 
-async function strFetch(){
-    const gson = new Gson();
-    const servletText = (await fetch('/strservlet')).text(); //get the servlet text
+async function strFetch(){
```
Comment (josueetcom) [inline portfolio/src/main/webapp/fetcher.js]: Horizontal whitespace is needed "before any open curly brace (`{`)"

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

Comment (josueetcom) [reply portfolio/src/main/webapp/fetcher.js]: Also, "On both sides of the double slash (`//`) that begins an end-of-line comment."

File: portfolio/src/main/webapp/fetcher.js [hunk 5]
```diff
@@ -1,9 +1,10 @@
 //quick fetch function 
-async function strFetch(){
-    const gson = new Gson();
-    const servletText = (await fetch('/strservlet')).text(); //get the servlet text
+async function strFetch(){  
+  const servletText = await (await fetch('/strservlet')).json(); //get the servlet text as json
```
Comment (josueetcom) [inline portfolio/src/main/webapp/fetcher.js]: Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

File: portfolio/src/main/webapp/fetcher.js (hunk 3 above)
Comment (josueetcom) [inline portfolio/src/main/webapp/fetcher.js]: Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

File: portfolio/src/main/webapp/fetcher.js [hunk 6]
```diff
@@ -1,9 +1,10 @@
...
 
-    const servletJSON = gson.toJSON(servletText) 
+  let randomQuote = ""
+  let randomIdx = Math.floor(Math.random() * servletText.length)
+  randomQuote = await servletText[randomIdx]; //choose a quote from a random index
 
-    (document.getElementById('strservletText')).innerText = servletText; //input our text into the HTML
+  (document.getElementById('strservletText')).innerText = randomQuote; //input our text into the HTML
```
Comment (josueetcom) [inline portfolio/src/main/webapp/fetcher.js]: Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

File: portfolio/src/main/java/com/google/sps/servlets/StringServlet.java [hunk 7]
```diff
@@ -19,9 +19,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+        quotations.add("Don't count the days, make the days count. -Muhammad Ali");
 
         /*
         //print quotations
```
Comment (josueetcom) [inline portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

File: portfolio/src/main/java/com/google/sps/servlets/StringServlet.java [hunk 8]
```diff
@@ -32,6 +32,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         //return the arraylist as json
```
Comment (josueetcom) [inline portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

File: portfolio/src/main/java/com/google/sps/servlets/StringServlet.java [hunk 9]
```diff
@@ -19,9 +19,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //create a new array list and input quoataions.
```
Comment (josueetcom) [inline portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: Horizontal whitespace is needed "On both sides of the double slash (`//`) that begins an end-of-line comment."

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

**Review by josueetcom — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/StringServlet.java [hunk 10]
```diff
@@ -8,30 +8,32 @@
...
-        //create a new array list and input quoataions.
+        // create a new array list and input quoataions.
         ArrayList<String> quotations = new ArrayList<String>();
 
-        quotations.add("<p>The past is not dead. In fact, it's not even past.  -William Faulkner</p>");
-        quotations.add("<p>Education is the passport to the future, for tomorrow belongs to those who prepare for it today. -Malcolm X</p>");
-        quotations.add("<p>Don't count the days, make the days count. -Muhammad Ali</p>");
+        quotations.add("The past is not dead. In fact, it's not even past.  -William Faulkner");
+        quotations.add("Education is the passport to the future," 
+                          +" for tomorrow belongs to those who prepare for it today. -Malcolm X");
```
Comment (josueetcom) [inline portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: nit: Need a space between operators too. Also the line continuation seems off (should be indented by 4 spaces from the previous line).

**Review by josueetcom — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/StringServlet.java (hunk 10 above)
Comment (jpaints3) [reply portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: Ok, gotcha. Just fixed those and will merge.

File: portfolio/src/main/java/com/google/sps/servlets/StringServlet.java [hunk 11]
```diff
@@ -21,7 +21,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         quotations.add("The past is not dead. In fact, it's not even past.  -William Faulkner");
         quotations.add("Education is the passport to the future," 
-                          +" for tomorrow belongs to those who prepare for it today. -Malcolm X");
+                              + " for tomorrow belongs to those who prepare for it today. -Malcolm X");
```
Comment (josueetcom) [inline portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: I was actually referring to this :)
```java
        quotations.add("Education is the passport to the future," 
            + " for tomorrow belongs to those who prepare for it today. -Malcolm X");
```
or this
```java
        quotations.add("Education is the passport to the future," +
            " for tomorrow belongs to those who prepare for it today. -Malcolm X");
```

Comment (jpaints3) [reply portfolio/src/main/java/com/google/sps/servlets/StringServlet.java]: Ah, haha, ok I'll fix that now

**Pull request merged**